apps/discovery_streams/mix.exs

```diff
@@ -43,7 +43,7 @@ defmodule DiscoveryStreams.Mixfile do
       {:credo, "~> 1.0", only: [:dev, :test], runtime: false},
       {:definition_kafka, in_umbrella: true},
       {:divo_kafka, "~> 0.1.6", only: [:integration]},
-      {:divo, "~> 1.1", only: [:dev, :integration]},
```

Comment:
same as alchemist, missing mix increment of app version number. since divo was updated in disc_streams, disc_streams needs a new mix version